dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/action/EditContentletAction.java

```diff
@@ -2891,14 +2892,17 @@ public void run() {
 
 			public void reindex() throws DotContentletStateException, DotStateException, DotSecurityException, DotDataException {
 				//DistributedJournalAPI jAPI = APILocator.getDistributedJournalAPI();
+
+				final ContentletCache contentletCache = CacheLocator.getContentletCache();
+
 				int count = 0;
-				for(String inode  : inodes){
+				for(final String inode  : inodes){
 
-					Contentlet contentlet = new Contentlet();
 					try{
-						contentlet = conAPI.find(inode, APILocator.getUserAPI().getSystemUser(), false);
+						final Contentlet contentlet = conAPI.find(inode, APILocator.getUserAPI().getSystemUser(), false);

```

Comment:
I have found an issue today with @erickgonzalez 
the api is returning mutable contentlet from the cache, we should do a copy of the object when you are going to change the properties, not sure if it is your case but we did something like:
```
final Contentlet contentlet = new Contentlet().getMap().putAll(conAPI.find(inode, APILocator.getUserAPI().getSystemUser(), false));
```